mobile-browser-based-version/src/helpers/training/training_manager.js

```diff
@@ -179,32 +235,38 @@ export class TrainingManager {
     });
   }
 
-  async _trainDistributed() {
-    const info = this.task.trainingInformation;
+  else {
+    console.log("Fast training");
 
-    await this.model.fit(this.data, this.labels, {
-      initialEpoch: 0,
+    let resized_data = this.data;
+
+    if(info.resize) {
```

Comment:
don't forget to lint your files before pushing :) we still don't have any pre-commit hooks, so you'll have to manually use ESLint